mon-pix/app/templates/components/form-textfield-date.hbs

```diff
@@ -1,105 +1,104 @@
-            class="form-textfield-icon__state form-textfield-icon__state--error"
-            alt={{t "common.form.error"}}
-          >
-        {{else}}
-          <img src="/images/icons/icon-success.svg"
-               class="form-textfield-icon__state form-textfield-icon__state--success"  alt={{t "common.form.success"}}>
+  <div class="form-textfield__date">
+    <div class="form-textfield__input-field-container {{if @disabled "form-textfield__input-container--disabled" this.dayInputContainerStatusClass}}">
+      <Input @type="text"
+             id={{@dayTextfieldName}}
+             name={{@dayTextfieldName}}
+             @value={{@dayInputBindingValue}}
+             {{on "focusout" (fn @onValidateDay @dayTextfieldName @dayInputBindingValue)}}
+             class="form-textfield__input {{this.dayInputValidationStatus}}"
+             ariaDescribedBy={{@dayValidationMessage}}
+             required={{@require}}
+             disabled={{@disabled}}
+             placeholder="JJ" />
+      <div class="form-textfield__icon">
+        {{#if this.dayHasIcon}}
+          {{#if (eq @dayValidationStatus 'error') }}
+            <img
+              src="/images/icons/icon-error.svg"
+              class="form-textfield-icon__state form-textfield-icon__state--error"
+              alt={{t "common.form.error"}}
+            >
+          {{else}}
+            <img src="/images/icons/icon-success.svg"
+                 class="form-textfield-icon__state form-textfield-icon__state--success"  alt={{t "common.form.success"}}>
+          {{/if}}
         {{/if}}
-      {{/if}}
+      </div>
     </div>
-  </div>
 
-  <div class="form-textfield__input-field-container {{if disabled "form-textfield__input-container--disabled" monthInputContainerStatusClass}}">
-    <Input @type="text"
-           @id={{monthTextfieldName}}
-           @name={{monthTextfieldName}}
-           @value={{monthInputBindingValue}}
-           @focus-out={{action onValidateMonth monthTextfieldName}}
-           class="form-textfield__input {{monthInputValidationStatus}}"
-           @autocomplete={{autocomplete}}
-           @ariaDescribedBy="monthValidationMessage"
-           required={{require}}
-           @disabled={{disabled}}
-           placeholder="MM" />
-    <div class="form-textfield__icon">
-      {{#if monthHasIcon}}
-        {{#if (eq monthValidationStatus 'error') }}
-          <img
-            src="/images/icons/icon-error.svg"
-            class="form-textfield-icon__state form-textfield-icon__state--error"
-            alt={{t "common.form.error"}}>
-        {{else}}
-          <img src="/images/icons/icon-success.svg"
-               class="form-textfield-icon__state form-textfield-icon__state--success"  alt={{t "common.form.success"}}>
+    <div class="form-textfield__input-field-container {{if @disabled "form-textfield__input-container--disabled" this.monthInputContainerStatusClass}}">
+      <Input @type="text"
+             id={{@monthTextfieldName}}
```

Comment:
Je pense qu'il faut garder le @ devant id : https://guides.emberjs.com/release/components/built-in-components/#toc_setting-attributes-on-input